packages/synapse-react-client/.storybook/main.mts

```diff
@@ -26,18 +26,17 @@ const config: StorybookConfig = {
   },
 
   typescript: {
-    // These options are from https://storybook.js.org/blog/material-ui-in-storybook/
-    check: false,
-    checkOptions: {},
+    // These options are from https://storybook.js.org/recipes/@mui/material
     reactDocgen: 'react-docgen-typescript',
     reactDocgenTypescriptOptions: {
-      // @ts-expect-error - speeds up storybook build time
-      allowSyntheticDefaultImports: false,
-      // speeds up storybook build time
-      esModuleInterop: false,
-      // makes union prop types like variant and size appear as select controls
+      // Speeds up Storybook build time
+      compilerOptions: {
```

Comment:
Only change seems to be nesting these in `compilerOptions`. Without this, Storybook would crash on startup.